articles/relnotes/index.md

```diff
@@ -16,6 +16,19 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.x
+
+*Release date: October 30th, 2019*
+
+This release contains the following:
+
+- IQ# and Python host functionality can now be installed using conda
+- Updates to Visual Studio Code & Visual Studio extensions to deploy language server as a self-contained executable, no longer requiring the .NET Core SDK to be installed  
```

Comment:
The "no longer requiring the .NET Core Sdk to be installed" should be replace by "eliminating the .NET Core Sdk version dependency" (meaning any version will work - which had been the problem before when people needed to update their dotnet version every time we updated ours), but they still need something installed.